azure-monitor-opentelemetry-distro/tests/configuration/test_configure.py

```diff
@@ -50,14 +54,18 @@ def test_configure_opentelemetry(
         exporter_mock.return_value = exp_init_mock
         resource_init_mock = Mock()
         resource_mock.create.return_value = resource_init_mock
+        sampler_init_mock = Mock()
+        sampler_mock.return_value = sampler_init_mock
         bsp_init_mock = Mock()
         bsp_mock.return_value = bsp_init_mock
-        configure_opentelemetry(
+        configure_azure_monitor(
             connection_string="test_cs",
```

Comment:
I think this conn str no longer works, right?

Reply:
Moved this to the exporter test.